rust/agents/validator/src/settings.rs

```diff
@@ -111,29 +111,31 @@ impl FromRawConf<'_, RawValidatorSettings> for ValidatorSettings {
                 .take_err(&mut err, || cwp + "chains" + &origin_chain_name)
         });
 
-        err.into_result()?;
-
-        let mut base = base.unwrap();
-        let origin_chain = origin_chain.unwrap();
-        let validator = validator.unwrap();
+        cfg_unwrap_all!(
+            cwp,
+            err: base,
```

Comment:
this macro syntax makes it seem like `base` is the only argument that has to do with `err`. How about something like this:
```rust
cfg_unwrap_all!(
            cwp,
            err: [
                base,
                origin_chain,
                validator,
                checkpoint_syncer,
                reorg_period
            ]
        );
        ```